scripts/netContain/scripts/contivNet.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
```

Comment:
This script has some minor changes made to it from what I recall. Git refused to provide a diff for it, no matter how much effort I've put into making it do so (e.g. rename + changes).